polyply/src/generate_templates.py

```diff
@@ -254,12 +254,12 @@ class GenerateTemplates(Processor):
     in the templates attribute. The processor also stores the volume
     of each block in the volume attribute.
     """
-    def __init__(self, *args, **kwargs):
+    def __init__(self, max_opt):
         super().__init__()
```

Comment:
Processor doesn't take arguments now, but I can't guarantee it won't in the future. Passing `*args` and `**kwargs` is basically free.